src/drivers/servo.cpp

```diff
@@ -3,11 +3,9 @@
 #include "servo.h"
 
 pwmcnt_t degreesToTicks(PWMDriver *driver, int angle) {
-	/*
-	 * 3600 degrees represent a whole period
-	 * Conversion is servo_width = pwm_width * 3 / 2 / 20
-	 */
-	return PWM_FRACTION_TO_WIDTH(driver, 3600, angle * 15 / 20);
+	/* 180 is degrees spread across 1000 to 2000 microseconds */
```

Comment:
hitec servos go from 900μs to 2100μs: [1](http://users.ece.utexas.edu/~valvano/EE345M/Servomanual.pdf) [2](https://servodatabase.com/servo/hitec/hs-422) [3](https://hitecrcd.com/faqs/servos/general-servos)

Reply:
That would help explain what we saw yesterday. I updated the last commit for that range